doc/classes/Curve3D.xml

```diff
@@ -140,6 +152,19 @@
 				Cubic interpolation tends to follow the curves better, but linear is faster (and often, precise enough).
 			</description>
 		</method>
+		<method name="interpolate_baked_up_vector" qualifiers="const">
+			<return type="Vector3">
+			</return>
+			<argument index="0" name="offset" type="float">
+			</argument>
+			<argument index="1" name="apply_tilt" type="bool" default="false">
+			</argument>
+			<description>
+				Returns an up vector within the curve at position "offset", where "offset" is measured as a distance in 3D units along the curve.
+				To do that, it finds the two cached up vectors where the "offset" lies between, then interpolates the values. If "apply_tilt" is [code]true[/code], an interpolated tilt is applied to the interpolated up vector.
+				If the curve has no up vectors, the function sends an error to the console, and returns (0, 1, 0).
```

Comment:
add [code] [/code] for offset, apply_tilt

Reply:
Should I do the same thing to interpolate_baked as well?